db/migrate/20201019154701_delete_lingering_subscription_contents.rb

```diff
@@ -0,0 +1,5 @@
+class DeleteLingeringSubscriptionContents < ActiveRecord::Migration[6.0]
+  def change
+    SubscriptionContent.where("created_at < ?", 1.week.ago).delete_all
```

Comment:
This seems a bit loose with a relative date range. I.e the migration run today on one machine is different to that run another a couple of days later. It'd be clearer if this was an absolute time.

Reply:
Sure 👍 I've changed it to the day after the fix was deployed which includes all lingering subscription contents.